.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-ci@0.0.13
+      with:
+        package-name: |
+          sros2
+          sros2_cmake
+    - name: Upload Logs
```

Comment:
Total noob questions, where are logs uploaded? How to access them? Who can access them?

Reply:
> Total noob questions

We're all discovering here :)

>  where are logs uploaded?

They will be uploaded on that job run. So going to that run's page on github should give you access to the artifacts. e.g. https://github.com/mikaelarguedas/sros2/actions/runs/58398774

> Who can access them?

Everyone I believe